### PR TITLE
Updated URLs. Other tidying.

### PR DESCRIPTION
**Pull request by Peppernrino**
Added link to BOINC site.
SSL'd main site.
Updated Tags URL.
Removed duplicate sentence.
Grammar.
Capitalized Freenode and IRC.
Updated IRC channel to #gridcoin-help, as it is the help channel and you have to be a registered user to use #gridcoin.
SSL'd Slack URL.
Fixed header line markdown.
Removed unnecessary spaces.
Another image file reduction! Saved ~3kB.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,36 +22,31 @@ These dependencies are required:
...
 For more detailed and platform-specific instructions, see [the doc folder.](doc/)
 
-        
 
-Development process
-===========================
 
-The master branch is regularly built and tested, but is not guaranteed 
-to be completely stable. [Tags](https://github.com/gridcoin/Gridcoin-Research/tags)
-are created regularly to indicate new official, stable release versions
```
Comment (barton2526) [inline README.md]: Why is this segment removed?

**Review by tomasbrod — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -22,36 +22,31 @@ These dependencies are required:
...
 if they don't match the project's coding conventions (see [coding.txt](doc/coding.txt))
 or are controversial.
 
 The master branch is regularly built and tested, but is not guaranteed
-to be completely stable. [Tags](https://github.com/gridcoin/Gridcoin-Research/tags) are regularly created to indicate new
+to be completely stable. [Tags](https://github.com/gridcoin-community/Gridcoin-Research/tags) are regularly created to indicate new
 stable release versions of Gridcoin.
 
 Feature branches are created when there are major new features being
 worked on by several people.
```
[the remark this answers is not in the document]
Comment (Peppernrino) [reply README.md]: I'm sorry you don't like how it looks in code, but of course it works! lol why would I submit something that doesn't work? :D 

I'm not sure what you mean by the semantic of branches... I only changed the URL to point to the community repo.

**Comment by Peppernrino** [comment]
I'm sorry you don't like how it looks in code, but of course it works! lol why would I submit something that doesn't work? :D

I'm not sure what you mean by the semantic of branches... I only changed the URL to point to the community repo.

**Review by tomasbrod — CHANGES_REQUESTED** [review]
- I retract my comment about branch semantics. You only changed URLs.
- However I stand by the dashes&equals. Please undo the regression. Markdown is a format that should look good both in source and when rendered. Onle lone equal sign under heading looks wrong. 
- I am not hating your work.

**Comment by Peppernrino** [comment]
updated in #1337 :)

**Pull request closed without merging**